source/change-streams/tests/unified/change-streams.yml

```diff
@@ -177,3 +177,163 @@ tests:
                 comment: "comment"
               commandName: getMore
               databaseName: *database0
+
+  - description: "Test unknown operationType MUST NOT err"
+    runOnRequirements:
+      - minServerVersion: "3.6"
```

Comment:
It doesn't look like _any_ of the tests in this file specify a `minServerVersion` below 3.6 (some use a greater version). If you'd like to clean this up, I'd propose removing `minServerVersion: "3.6"` throughout the file (both in your new tests and the few existing ones that shared it) and just adding this to the top-level `runOnRequirements`.